src/ainft.ts

```diff
@@ -141,12 +148,21 @@ export default class AinftJs {
     this.setSigner(signer);
   }
 
+  async open() {
```

Comment:
could you confirm that the code under examples/ and sample/ is compatible with these changes?

Reply:
@platfowner I will update code in `examples`!
but i think it would be okay to delete the `sample`since its contents are already included in the `examples`, so what do you think about removing the `sample` code?

Reply:
👌

Reply:
Thank you for comment!